pom.xml

```diff
@@ -33,6 +33,19 @@
         <java.level>8</java.level>
         <jenkins.version>2.289.1</jenkins.version>
     </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.jenkins.tools.bom</groupId>
+                <artifactId>bom-2.289.x</artifactId>
+                <version>1008.vb9e22885c9cf</version>
```

Comment:
This version number is pretty wild :) 

If you want to use a version number from 'jenkins-incremental' reposiotory, then the latest one is 1084.va39f094ca02f (https://mvnrepository.com/artifact/io.jenkins.tools.bom/bom-2.289.x?repo=jenkins-incrementals).

However, I personally would rely on more stable versions from jenkins-releases, the latest is 1055.v9637a690f734
(https://mvnrepository.com/artifact/io.jenkins.tools.bom/bom-2.289.x?repo=jenkins-releases)

What do you think?

Reply:
:)

That's a version number generated by the continuous delivery process used by JEP-229.

The release detection logic on https://mvnrepository.com seems to suffer the same issue as Apache Maven does.  It does not detect some of the official releases as stored at https://repo.jenkins-ci.org/ui/repos/tree/General/releases/io/jenkins/tools/bom/bom-2.289.x/ .

I believe the reason it does not detect those releases is described in https://github.com/jenkinsci/incrementals-tools/pull/24.  That should be fixed for future releases, since the [version number string generator 1.3](https://github.com/jenkinsci/incrementals-tools/releases/tag/parent-1.3) has been modified to adapt to the naming rules used by Apache Maven.